CMakeLists.txt

```diff
@@ -226,6 +226,18 @@ if(BUILD_MAYAUSD_LIBRARY)
     endif()
 endif()
 
+if(USD_VERSION VERSION_GREATER_EQUAL "0.24.11")
+    # In USD v24.11 Pixar USD has completely removed Boost.
+    # However MayaUsd is still using a few of the Boost components, so
+    # when using USD v24.11 (or greater) we need to find Boost ourselves
+    # (from the USD build, since Boost is still required for OpenImageIO).
+    if(NOT DEFINED ENV{BOOST_ROOT})
+        set(ENV{BOOST_ROOT} ${PXR_USD_LOCATION})
+    endif()
+
+    find_package(Boost REQUIRED COMPONENTS headers)
```

Comment:
With USD v24.11 we only need boost headers which we must find ourselves, since USD no longer has Boost as a dependency (and thus not in the USD cmake files in our USD artifact).